src/components/dialogs/connectivity-edition.js

```diff
@@ -198,51 +216,136 @@ export const useConnectivityValue = ({
                 busOrBusbarSection: null,
                 connectionDirection: null,
                 connectionName: null,
+                connectionPosition: null,
             };
 
         const ret = {
             voltageLevel: ided(voltageLevelObjOrId),
             busOrBusbarSection: ided(busOrBusbarSectionObjOrId),
             connectionDirection: ided(connectionDirection),
             connectionName: ided(connectionName),
+            connectionPosition: ided(connectionPosition),
         };
         return ret;
     }, [
         voltageLevelObjOrId,
         busOrBusbarSectionObjOrId,
         connectionDirection,
         connectionName,
+        connectionPosition,
     ]);
 
+    const [open, setOpen] = React.useState(false);
+
+    const handleClickOpen = () => {
+        setOpen(true);
+    };
+
+    const handleClose = () => {
+        setOpen(false);
+    };
+
     const render = useMemo(() => {
         return (
-            <Grid container direction={direction || 'row'} spacing={2}>
-                <Grid item xs={gridSize} align="start">
-                    {voltageLevelField}
-                </Grid>
-                <Grid item xs={gridSize} align="start">
-                    {busOrBusbarSectionField}
+            <>
+                <Grid container direction={direction || 'row'} spacing={2}>
+                    <Grid
+                        item
+                        xs={withPosition && withDirectionsInfos ? 4 : gridSize}
+                        align="start"
+                    >
+                        {voltageLevelField}
+                    </Grid>
+                    <Grid
+                        item
+                        xs={withPosition && withDirectionsInfos ? 4 : gridSize}
+                        align="start"
+                    >
+                        {busOrBusbarSectionField}
+                    </Grid>
+
+                    {withDirectionsInfos && (
```

Comment:
Why use the diamond twice, instead of merging the two tests in one ?

Reply:
it's a temporary condition , i need to have each block tested independently, for the scope of this pr we add the position field just for injections (mcs, load, generator) , later we gonna change this to add position field to injections , branches and twt...